apps/jobs/models.py

```diff
@@ -162,3 +162,26 @@ def save(self, *args, **kwargs):
 
         submission_instance = super(Submission, self).save(*args, **kwargs)
         return submission_instance
+
+    @property
+    def submissions_count_on_a_challenge_phase(self):
+        # To get total submissions count on a challenge phase
+        submission = Submission.objects.filter(challenge_phase=self.challenge_phase,
```

Comment:
Also variable name should go in sync with what it is for. Here submission_count.

Please update variable names in all the properties.